doc/modules/reference.rst

```diff
@@ -142,6 +165,17 @@ uses.
    NiftiMapsMasker
    NiftiSpheresMasker
 
+**Functions**:
+
+.. currentmodule:: nilearn.input_data
+
+.. autosummary::
+   :toctree: generated/
+   :template: function.rst
+
+   filter_and_extract
+   filter_and_mask
```

Comment:
I don't think that we want to expose these two functions: to me, they are internal.